db/models/User.js

```diff
@@ -1,9 +1,6 @@
 import { Schema, model } from 'mongoose';
 
-const SchemaTypes = Schema.Types;
-
 const userSchema = new Schema({
-  _id: SchemaTypes.ObjectId,
   username: String,
   password_hashed: String,
   passsword_salt: String,
```

Comment:
Typo in `passsword_salt`

Reply:
https://github.com/EasyEventV2/ee2_api/pull/4/commits/721a6eed7f0d09d0bab1b851892f16ed67b3c9ba
removed